CHANGELOG.rst

```diff
@@ -14,13 +14,21 @@ The changes listed in this file are categorised as follows:
     - Fixed: any bug fixes
     - Security: in case of vulnerabilities.
 
+feature/parallel-process-common
```

Comment:
All these changes can be deleted because they're already below (we put them below master in anticipation of things being merged).

Removing these lines will also fix the CI